contracts/base/BaseAccount.sol

```diff
@@ -23,14 +25,10 @@ import { IBaseAccount } from "../interfaces/base/IBaseAccount.sol";
 /// @author @filmakarov | Biconomy | filipp.makarov@biconomy.io
 /// @author @zeroknots | Rhinestone.wtf | zeroknots.eth
 /// Special thanks to the Solady team for foundational contributions: https://github.com/Vectorized/solady
-contract BaseAccount is IBaseAccount {
+contract BaseAccount is Storage, IBaseAccount {
```

Comment:
??

Reply:
I moved the _ENTRYPOINT to storage, as it needs to be used by both BaseAccount and ModuleManager for the new `onlyAuthorized`

Reply:
why would you do that?
it's not a storage
it's immutable : address internal immutable _ENTRYPOINT;

what's new onlyAuthorized for?

Reply:
onlyAuthorized is checking msg.sender is EP, self or Executor while other modifier are exclusively checking EP EP+self or Executor only

if the naming is a problem we can change it